enterprise/cmd/executor/internal/run/validate.go

```diff
@@ -71,6 +76,26 @@ func RunValidate(cliCtx *cli.Context, logger log.Logger, config *config.Config)
 	return nil
 }
 
+var AuthorizationFailedErr = errors.New("failed to authorize with frontend, is the authorization token set correctly?")
```

Comment:
```suggestion
var AuthorizationFailedErr = errors.New("failed to authorize with frontend, is executors.accessToken set correctly in the site-config?")
```

I think publicly this is only ever referred to as the `accessToken` so calling it an authorization token may be a little confusing.